cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -38,9 +41,18 @@ public class KafkaCruiseControlConfig extends AbstractConfig {
   private static final ConfigDef CONFIG;
 
   static {
-    CONFIG = CruiseControlRequestConfig.define(CruiseControlParametersConfig.define(AnomalyDetectorConfig.define(
-        AnalyzerConfig.define(ExecutorConfig.define(MonitorConfig.define(WebServerConfig.define(
-            UserTaskManagerConfig.define(new ConfigDef())))))))).withClientSslSupport().withClientSaslSupport();
+    ConfigDef configDef = new ConfigDef();
```

Comment:
Changes in this file look like a refactoring rather than part of the feature, can this be extracted in a separate PR?